swift-build.sh

```diff
@@ -46,9 +34,19 @@ build() {
         -DCMAKE_Swift_COMPILER="${ANDROID_SDK}/usr/bin/swiftc" \
         -DCMAKE_Swift_COMPILER_FORCED=TRUE \
         -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=${LIBRARY_OUTPUT_DIRECTORY} \
-        ${PROJECT_DIRECTORY}
+        -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE \
+        -S ${PROJECT_DIRECTORY} \
+        -B ${BUILD_DIR}
+    
+    echo "Finished configure ${CMAKE_BUILD_TYPE} for ${ANDROID_ABI}"
+}
+
+build() {
+    # reconfigure when build dir does not exist empty
+    [[ -d ${BUILD_DIR} ]] || configure
```

Comment:
Nice. How’d you get this working in the end?

Reply:
We still tried to cmake —build . (In current directory) without cding into. Changing it to cmake —build $BUILD_DIR did the trick. @ephemer